ejb/src/test/java/biz/karms/sinkit/ejb/impl/ArchiveServiceEJBTest.java

```diff
@@ -0,0 +1,108 @@
+package biz.karms.sinkit.ejb.impl;
+
+import biz.karms.sinkit.ejb.ArchiveService;
+import biz.karms.sinkit.ejb.BlacklistCacheService;
+import biz.karms.sinkit.ejb.WhitelistCacheService;
+import biz.karms.sinkit.ejb.cache.pojo.WhitelistedRecord;
+import biz.karms.sinkit.ejb.elastic.ElasticService;
+import biz.karms.sinkit.ejb.elastic.logstash.LogstashClient;
+import biz.karms.sinkit.ejb.util.IoCIdentificationUtils;
+import biz.karms.sinkit.exception.TooOldIoCException;
+import biz.karms.sinkit.ioc.*;
+import org.elasticsearch.index.query.QueryBuilder;
+import org.elasticsearch.search.sort.SortBuilder;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.logging.Logger;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+/**
+ * @author Tomas Kozel
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ArchiveServiceEJBTest {
+
+    //TODO: Shouldn't log messages be also shown (could be useful for debugging)
+    @Mock
+    private Logger log;
+
+    @Mock
+    private ElasticService elasticService;
+
+    @Mock
+    private LogstashClient logstashClient;
+
+    @InjectMocks
+    private ArchiveServiceEJB archiveService;
+
+
+/**
+ * Blacklist tests
+ */
+    @Test
+    public void deactivateRecordTest() throws Exception{
+
+        IoCRecord ioc= getIoCRecord(null, "facebook.malware.com","SomeFeed",
```

Comment:
@hugeox Refrain from using branded names, please; use "example" etc.  If you see such occurrences in the older code, fix them please.